src/webpack.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { IgnorePlugin } from 'webpack';
```

Comment:
Unfortunately this means we have yet another implicit dependency :-/

Theoretically it's a safe one, if someone's importing the "webpack" entry point they're likely to have webpack available, but there's no way to declare version compatibility.